### ISSUE TITLE
 [Request] Default quantity of tokens when reisuing tokens to 0

### ISSUE THREAD

**Issue by jeroz1** [RavenProject/Ravencoin#451]
Users report that they accidentally issue an extra token when they only want to update the IPFS hash.

**Comment by blondfrogs** [comment]
This is a good change. I will add this into the next update

**Comment by blondfrogs** [comment]
Will be added once #456 is merged into develop @jeroz1

**Comment by jeroz1** [comment]
Great, thanks!
